src/VisualStudio/LiveShare/Test/FindAllReferencesHandlerTests.cs

```diff
@@ -108,17 +110,17 @@ public async Task TestFindAllReferencesAsync_InvalidLocation()
 
         private static async Task<LSP.Location[]> RunFindAllReferencesAsync(Solution solution, LSP.Location caret, bool includeDeclaration)
         {
-            var request = new LSP.ReferenceParams()
+            var request = (JObject)JToken.FromObject(new LSP.ReferenceParams()
```

Comment:
Does this need a tracking bug to undo these? Since it's more weakly typed if things are still passing around object it might not be obvious.